src/app/qgsrasterlayerproperties.cpp

```diff
@@ -2043,6 +2051,27 @@ bool QgsRasterLayerProperties::rasterIsMultiBandColor()
   return mRasterLayer && nullptr != dynamic_cast<QgsMultiBandColorRenderer *>( mRasterLayer->renderer() );
 }
 
+void QgsRasterLayerProperties::updateInformationContent()
+{
+  // We are using QgsWebView and the renderer is completely different from QTextBrowser
+  // add some extra style here
+  QString myStyle = QgsApplication::reportStyleSheet();
+  myStyle.append( QStringLiteral( "body { margin: auto; width: 97% } "
+                                  "table.tabular-view, table.list-view { border-collapse: collapse; table-layout:fixed; width: 100% !important; } "
+                                  "h1 { line-height: inherit; } "
+                                  "td, th { word-wrap: break-word; vertical-align: top; } "
+                                  ".list-view th:first-child, .list-view td:first-child { width: 15%; } "
+                                  ".list-view.highlight { padding-left: inherit; } "
+                                  ".tabular-view th:first-child, .tabular-view td:first-child { width: 20%; } "
+                                  ".tabular-view th.strong { background-color: #eee; }"
+                                  ".tabular-view th, .tabular-view td { "
+                                  "  border: solid 1px #eee;"
+                                  "} \n " ) );
+  const QString html { mRasterLayer->htmlMetadata().replace( QStringLiteral( "<head>" ), QStringLiteral( R"raw(<head><style type="text/css">%1</style>)raw" ) ).arg( myStyle ) };
```

Comment:
What does raw do here?

Reply:
it's a raw string, I find it much more readable when there are double quotes.
I always use `raw` as a prefix/suffix but I'm open to suggestions.

Reply:
Ah - I've gotta admit, I've never used these and mistakenly thought this was some new js/html something which I'd never seen before!

Reply:
they are very handy especially when you have to deal with regular expressions: you don't have do escape backslashes or double quotes.